src/rpc/rpc-handler.js

```diff
@@ -335,7 +309,6 @@ module.exports = class RpcHandler {
   *
   * @return {String}
   */
- // eslint-disable-next-line
```

Comment:
this is needed because this is not used in the method

we can make it static if needed instead